apps/formbricks-com/app/docs/self-hosting/migration-guide/page.mdx

```diff
@@ -29,12 +29,16 @@ To run all these steps, please navigate to the `formbricks` folder where your `d
 <CodeGroup title="Backup Postgres">
 
 ```bash
-docker exec formbricks-quickstart-postgres-1 pg_dump -U postgres -d formbricks > formbricks_pre_v1.6_$(date +%Y%m%d_%H%M%S).sql
+docker exec formbricks-quickstart-postgres-1 pg_dump -Fc -U postgres -d formbricks > formbricks_pre_v1.6_$(date +%Y%m%d_%H%M%S).dump
```

Comment:
A note on why this change:

In a restore scenario sql files can only be restored with `psql` and require an empty database.

Using `-Fc` for *custom format* allows to use `pg_restore` (see below) to restore in one take including cleaning the database.